src/net/sourceforge/kolmafia/request/UneffectRequest.java

```diff
@@ -308,170 +307,170 @@ public static final void reset() {
     Set<Integer> removableEffects;
 
     removableEffects = new HashSet<Integer>();
-    removeWithItemMap.put(IntegerPool.get(ItemPool.ANTIDOTE), removableEffects);
-    removableEffects.add(IntegerPool.get(EffectPool.HARDLY_POISONED));
-    removableEffects.add(IntegerPool.get(EffectPool.MAJORLY_POISONED));
-    removableEffects.add(IntegerPool.get(EffectPool.A_LITTLE_BIT_POISONED));
-    removableEffects.add(IntegerPool.get(EffectPool.SOMEWHAT_POISONED));
-    removableEffects.add(IntegerPool.get(EffectPool.REALLY_QUITE_POISONED));
+    removeWithItemMap.put(ItemPool.ANTIDOTE, removableEffects);
+    removableEffects.add(EffectPool.HARDLY_POISONED);
+    removableEffects.add(EffectPool.MAJORLY_POISONED);
+    removableEffects.add(EffectPool.A_LITTLE_BIT_POISONED);
+    removableEffects.add(EffectPool.SOMEWHAT_POISONED);
+    removableEffects.add(EffectPool.REALLY_QUITE_POISONED);
 
     removableEffects = new HashSet<Integer>();
```

Comment:
Outside the scope of this PR, but HashSet<>(), everywhere.

Reply:
I will file a follow-up that just makes my IDE do it everywhere possible :P

Reply:
IDE has only an option for "in file", so let's keep going with the original method of fixing lints as we edit files containing them.